### ISSUE TITLE
[BLOCK] www.historisches-tonarchiv.de

### ISSUE THREAD

**Issue by SoftwareRat** [hagezi/dns-blocklists#4612]
### Which domain(s) should be blocked?

www.historisches-tonarchiv.de

### Why should these domain(s) be blocked?

The domain [www.historisches-tonarchiv.de](http://www.historisches-tonarchiv.de/) is currently a parked domain, which means it is not actively being used for a website or service. Parked domains often display placeholder content or advertisements, and they may not provide any meaningful information or resources to visitors.

### I confirm ...

- [X] that I have checked if there is no other issue for the domain(s) whereby they were unblocked or the blocking was declined.
- [X] that I have checked that the domain(s) are not already blocked.

### Privacy

- [X] I confirm that the report does not contain any private information.

**Comment by hagezi** [comment]
I do not block pure parked domains as long as they are not used to distribute ads. These only clutter up the lists and make them unnecessarily large. There are millions of parked domains.

**Comment by xRuffKez** [comment]
@jarelllama

**Comment by jarelllama** [comment]
I only keep track of parked domains that were once scam/malicious sites so I can add them back if they become unparked.
